spec/converters/json_spec.rb

```diff
@@ -0,0 +1,154 @@
+require 'spec_helper'
+require 'ronin/masscan/converters/json'
+require 'tempfile'
+
+RSpec.describe Ronin::Masscan::Converters::JSON do
+  let(:fixtures_path) { File.expand_path(File.join(__dir__, '..', 'fixtures')) }
```

Comment:
If we want to use separate files for the `Converters` specs, maybe move them into `spec/converters/fixtures/` that way the fixtures files are closer to the spec files that use them?